src/app-bridge/app-bridge.test.ts

```diff
@@ -232,4 +252,14 @@ describe("AppBridge", () => {
 
     window.location.href = currentLocationHref;
   });
+
+  it("dispatches 'notifyReady' action when created", (done) => {
+    appBridge = new AppBridge();
+
+    window.addEventListener("message", (event) => {
+      if (event.data.type === ActionType.notifyReady) {
+        done();
+      }
+    });
```

Comment:
Shouldn't event listener be defined first?
```suggestion
    window.addEventListener("message", (event) => {
      if (event.data.type === ActionType.notifyReady) {
        done();
      }
    });
    
    appBridge = new AppBridge();
```

Reply:
Can be, but postMessage is on the event queue so it did wait :)